docs/csharp/language-reference/keywords/ref.md

```diff
@@ -83,6 +84,8 @@ return ref DecimalArray[0];
 
 In order for the caller to modify the object's state, the reference return value must be stored to a variable that is explicitly defined as a [ref local](#ref-locals).
 
+The called method may also declare the return value as `ref readonly` to return the value by reference, and enforce that the calling code cannot modify the returned value. The calling method can avoid copying the returned valued by storing the value in a variable that is defined as a [ref readonly local](#ref-readonly-local).
```

Comment:
The last sentence gives an impression that `local` is a valid keyword and the following code compiles:
```csharp
ref readonly local myVar = ref MethodThatReturnsRefReadonly();
```

What about the following rephrasing:
>...by storing the value in a local [ref readonly](#ref-readonly-local] variable.